metadata_annotation_gdsc.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Everyone reading this notebook will have lost you at this point without an explanation! Now we have a PCA of drug response - what?? Unheard of, very novel, much confused</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
@yugeji Could you please give more details? It is about performing clustering on cell lines based on drug response data. I am not so sure whether it is correct to group cell lines like that way.